src/__tests__/components/app/Modal.test.js

```diff
@@ -0,0 +1,410 @@
+import React from "react";
+import * as Immutable from "immutable";
+import { mount, shallow, render } from "enzyme";
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import merge from 'merge';
+import { ShortcutProvider } from '../../../components/keyshortcuts/ShortcutProvider';
+
+import Modal from "../../../components/app/Modal";
+import fixtures from '../../../../test_setup/fixtures/modal.json';
+
+const mockStore = configureStore([]);
+
+
+const appHandlerState = {
```

Comment:
Can we export states from the handlers instead ? Same goes for keymap - it's already defined in constants, just needs a wrapper function that'd accept a `mod` key, and then we could reuse it in tests too.

Reply:
I've updated the PR pls check. There was however a missing export of the initiaState in src/reducers/appHandler.js (fixed that as well with the changes)